app/views/organizations/tasks/show.turbo_stream.erb

```diff
@@ -0,0 +1,3 @@
+<%= turbo_stream.update @task do %>
```

Comment:
Let's also create a `show.html.erb` to support the html response path as well. I think it can just be:
```erb
<%= render @task %>
```